libcontainer/rootfs_linux.go

```diff
@@ -25,14 +25,28 @@ import (
 
 const defaultMountFlags = syscall.MS_NOEXEC | syscall.MS_NOSUID | syscall.MS_NODEV
 
+// setupDev returns true if /dev needs to be set up.
+func needsSetupDev(config *configs.Config) (bool, error) {
+	setupDev := true
+	for _, m := range config.Mounts {
+		if m.Device == "bind" && m.Destination == "/dev" {
```

Comment:
I think we should check for `/dev/` as well since it is a valid input for destination.